07-routing-gossip.md

```diff
@@ -111,90 +118,99 @@ announcement message; this is accomplished by having a signature from each
-keys of the two nodes who are operating the channel, such that
-`node_id_1` is the numerically-lesser of the two DER-encoded keys sorted in
-ascending numerical order, and MUST set `bitcoin_key_1` and
-`bitcoin_key_2` to `funding_pubkey`s of `node_id_1` and `node_id_2`
-respectively.
-
-The creating node MUST compute the double-SHA256 hash `h` of the message, starting at offset 256, up to the end of the message.
-Thus the hash skips the 4 signatures, but hashes the rest of the message, including any future fields appended to the end.
-`node_signature_1` and `node_signature_2` MUST be valid signatures of the hash `h` using the secret associated with `node_id_1` and `node_id_2` respectively.
-`bitcoin_signature_1` and `bitcoin_signature_2` MUST be valid signatures of the hash `h` using the secret associated with `bitcoin_key_1` and `bitcoin_key_2` respectively.
-
-The creating node SHOULD set `len` to the minimum length required to
-hold the `features` bits it sets.
-
-The receiving node MUST verify the integrity and authenticity of the message by verifying the signatures.
-If there is an unknown even bit in the `features` field the receiving node MUST NOT parse the remainder of the message and MUST NOT add the channel to its local network view, and SHOULD NOT forward the announcement.
-
-The receiving node MUST ignore the message if the output specified
-by `short_channel_id` does not
-correspond to a P2WSH using `bitcoin_key_1` and `bitcoin_key_2` as
-specified in [BOLT #3](03-transactions.md#funding-transaction-output).
-The receiving node MUST ignore the message if this output is spent.
-
-The receiving node MUST ignore the message if the specified `chain_hash`
-is unknown to the receiver.
-
-Otherwise, the receiving node SHOULD fail the connection if
-`bitcoin_signature_1`, `bitcoin_signature_2`, `node_signature_1` or
-`node_signature_2` are invalid or not correct.
-
-Otherwise, if `node_id_1` or `node_id_2` are blacklisted, it SHOULD
-ignore the message.
-
-Otherwise, if the transaction referred to was not previously announced
-as a channel, the receiving node SHOULD queue the message for
-rebroadcasting, but MAY choose not to for messages longer than
-the minimum expected length. If it has previously received a valid
-`channel_announcement` for the same transaction in the same block, but
-for a different `node_id_1` or `node_id_2`, it SHOULD blacklist the
-previous message's `node_id_1` and `node_id_2` as well as this
-`node_id_1` and `node_id_2` and forget channels connected to them,
-otherwise it SHOULD store this `channel_announcement`.
-
-The receiving node SHOULD forget a channel once its funding output has
-been spent or reorganized out.
+The creating node:
+  - MUST set `chain_hash` to the 32-byte hash that uniquely identifies the chain
+  that the channel was opened within:
+    - MUST set the _Bitcoin blockchain_ `chain_hash` value (encoded in hex) to: `000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f`.
```

Comment:
It would probably be better to say in line 122 "MUST set its `chain_hash`" to better differentiate it from line 124, and then I'd suggest not indenting line 124, which suggests a dependency rather than a parallelism.

Reply:
changed